app/models/webchat.rb

```diff
@@ -0,0 +1,29 @@
+class Webchat
+  include ActiveModel::Validations
+
+  CONFIG_PATH = Rails.root.join("lib/webchat/config.yaml")
```

Comment:
I left it as global so that we don't hardcode it directly inside the function. This also keeps it cleaner and follows the standards set in other models https://github.com/alphagov/finder-frontend/blob/master/lib/brexit_checker/question.rb